tests/integration/pox/pox-3/reject-pox.spec.ts

```diff
@@ -0,0 +1,202 @@
+
+    await waitForNextRewardPhase(network, orchestrator, 1);
+
+    // Verify PoX is active
+    poxInfo = await getPoxInfo(network);
+    expect(poxInfo.current_cycle.is_pox_active).toBe(true);
+
+    // Chloe rejects pox, but does not have enough to de-activate it
+    response = await broadcastRejectPox({
+      poxVersion: 3,
+      network,
+      account: Accounts.WALLET_3,
+      fee,
+      nonce: chloeNonce++,
+    });
+    expect(response.error).toBeUndefined();
+    [block, tx] = await waitForStacksTransaction(orchestrator, response.txid);
+    expect(tx.success).toBeTruthy();
+
+    let isPoxActive = await callReadOnlyIsPoxActive(
+      3,
+      network,
+      Accounts.WALLET_1,
+      poxInfo.current_cycle.id
+    );
+    expect(isPoxActive).toEqual(trueCV());
+
+    isPoxActive = await callReadOnlyIsPoxActive(
+      3,
+      network,
+      Accounts.WALLET_1,
+      poxInfo.next_cycle.id
+    );
+    expect(isPoxActive).toEqual(trueCV());
+
+    poxInfo = await getPoxInfo(network);
+    expect(poxInfo.rejection_votes_left_required).toBeGreaterThan(0);
+  });
+
+  it("pox should be disabled if enough STX reject", async () => {
+    const network = new StacksTestnet({ url: orchestrator.getStacksNodeUrl() });
+
+    let response = await broadcastRejectPox({
+      poxVersion: 3,
+      network,
+      account: Accounts.FAUCET,
+      fee,
+      nonce: faucetNonce++,
+    });
+    expect(response.error).toBeUndefined();
+    let [block, tx] = await waitForStacksTransaction(
+      orchestrator,
+      response.txid
+    );
+    expect(tx.success).toBeTruthy();
+
+    let poxInfo = await getPoxInfo(network);
+    expect(poxInfo.rejection_votes_left_required).toBe(0);
+
+    let isPoxActive = await callReadOnlyIsPoxActive(
+      3,
+      network,
+      Accounts.WALLET_1,
+      poxInfo.current_cycle.id
+    );
+    expect(isPoxActive).toEqual(trueCV());
+
+    isPoxActive = await callReadOnlyIsPoxActive(
+      3,
+      network,
+      Accounts.WALLET_1,
+      poxInfo.next_cycle.id
+    );
+    expect(isPoxActive).toEqual(falseCV());
+
+    await waitForNextRewardPhase(network, orchestrator);
+
+    poxInfo = await getPoxInfo(network);
+    expect(poxInfo.rejection_votes_left_required).toBe(0);
+
+    isPoxActive = await callReadOnlyIsPoxActive(
+      3,
+      network,
+      Accounts.WALLET_1,
+      poxInfo.current_cycle.id
+    );
+    expect(isPoxActive).toEqual(trueCV());
+  });
+
+  it("pox should be re-enabled in the next cycle", async () => {
+    const network = new StacksTestnet({ url: orchestrator.getStacksNodeUrl() });
+
+    await waitForNextRewardPhase(network, orchestrator);
+
+    let poxInfo = await getPoxInfo(network);
+    expect(poxInfo.rejection_votes_left_required).toBeGreaterThan(0);
+
+    let isPoxActive = await callReadOnlyIsPoxActive(
+      3,
+      network,
+      Accounts.WALLET_1,
+      poxInfo.current_cycle.id
+    );
+    expect(isPoxActive).toEqual(falseCV());
```

Comment:
Why false here? Or is it enabled in the next cycle?

Reply:
See above. Fixed in c32a496